omod/src/main/webapp/fragments/field/personRelationship.gsp

```diff
@@ -27,7 +27,7 @@
         </p>
 
         <p class="left">
-            <input type="text" class="person-typeahead" placeholder="${ui.message('registrationapp.person.name')}"
+            <input type="text" class="person-typeahead" placeholder="${ui.message(ui.encodeHtmlContent('registrationapp.person.name'))}"
```

Comment:
I think `ui.encodeHtmlAttribute` might be a better fit here

Reply:
i agree 100%